casepro/backend/junebug.py

```diff
@@ -273,8 +277,13 @@ class JunebugBackend(BaseBackend):
     def __init__(self):
         self.identity_store = IdentityStore(
             settings.IDENTITY_API_ROOT, settings.IDENTITY_AUTH_TOKEN, settings.IDENTITY_ADDRESS_TYPE)
-        self.message_sender = JunebugMessageSender(
-            settings.JUNEBUG_API_ROOT, settings.JUNEBUG_CHANNEL_ID, settings.JUNEBUG_FROM_ADDRESS, self.identity_store)
+        self.message_senders = dict([
+            (channel_id, JunebugMessageSender(
```

Comment:
We might want to add some validation here, `JUNEBUG_CHANNELS` might be misconfigured.